Makefile

```diff
@@ -100,7 +100,7 @@ reinstall-p3:      ## Re-initialize the virtualenv with Python 3.x
 	PIP_CMD=pip3 VENV_OPTS="-p `which python3`" make install
 
 lint:              ## Run code linter to check code style
-	($(VENV_RUN); pep8 --max-line-length=120 --ignore=E128 --exclude=node_modules,$(VENV_DIR)*,dist .)
+	($(VENV_RUN); flake8 --max-line-length=120 --ignore=E128 --exclude=node_modules,$(VENV_DIR)*,dist .)
```

Comment:
add __--show-source__ to the flake8 command to see the source code context of the error.

The blocking issue is #297